lib/annotate.rb

```diff
@@ -136,4 +131,13 @@ def self.bootstrap_rake
     load_tasks
     Rake::Task[:set_annotation_options].invoke
   end
+
+  class << self
```

Comment:
This seems interesting, I'm not sure I've seen this before. Could you explain what is supposed to be happening? 

Seems like a module that has class-level methods. But I'm confused as to what happens when doing the
```ruby
class << self
...
end
```

in a module

Reply:
@drwl when you want to defined private method, you can write like this.

https://stackoverflow.com/questions/4952980/how-to-create-a-private-class-method/12311586#12311586

Reply:
Gotcha, thanks for the link